mlflow/spark.py

```diff
@@ -59,7 +59,7 @@ def save_model(spark_model, path, mlflow_model=Model(), conda_env=None, jars=Non
             str(type(spark_model))))
     if not isinstance(spark_model, PipelineModel):
         raise Exception("Not a PipelineModel. SparkML can currently only save PipelineModels.")
-    spark_model.save(os.path.join(path, "model"))
+    spark_model.save("file:" + os.path.join(path, "model"))
```

Comment:
I think this might break if `path` is a relative path - seems Spark can either handle "file:/" or "file:///" URIs (where the latter seems to follow the format of a [file URI](https://en.wikipedia.org/wiki/File_URI_scheme) pointing to the local filesystem).

To be safe / allow users to pass either a relative or absolute path to `path` we could use `"file:///" + os.path.join(path, "model")`, or just validate that `path` is an absolute path.

Reply:
+1